typescript/sdk/src/chain-metadata.ts

```diff
@@ -17,6 +17,21 @@ export const ethereum: ChainMetadata = {
   finalityBlocks: 7,
 };
 
+export const arbitrum: ChainMetadata = {
+  id: 0x617262, // b'arb' interpreted as an int
+  finalityBlocks: 1,
```

Comment:
We probably don't have a good setup here, but is that how rollup finality works?

Reply:
This ones tougher - will have to do some reading, but any insight here would be helpful. I'm no expert but I assume we are trusting the RPC endpoint to be accurate and not falling for any fraud, in which case I'm pretty sure we don't need to worry about reorgs. Definitely fuzzy here though and will read some more before doing the agent deploy

Reply:
https://developer.offchainlabs.com/docs/finality is the relevant page. I think long story short, we should just assume instant finality, everything else is going to be too complicated